packages/astro/src/core/build/generate.ts

```diff
@@ -223,6 +223,9 @@ async function generatePage(
 		styles,
 		mod: pageModule,
 	};
+
+	const maxPathsToLog = 100;
```

Comment:
You might want to disable this when verbose mode is enabled. I'm not 100% sure how you get it from here, but a search for `verbose` should hopefully find an example

Reply:
Done!